lib/projects/addCredits.ts

```diff
@@ -0,0 +1,76 @@
+import { NetworkType, getNetworkConfig } from './utils/getNetworkConfig'
+import CreditManagerAbi from '../abis/CreditManager.json'
+import arg from 'arg';
+import { PROJECT_INFO } from './Projects';
+
+if (!process.env.PRIVATE_KEY) {
+  throw new Error('PRIVATE_KEY environment variable is not set');
+}
+
+
+const getCreditInfo = (creditId: string) => {
+  const info = PROJECT_INFO.find(projectInfo => `${projectInfo.projectId}-${projectInfo.vintage}` === creditId)
```

Comment:
ah ok, nice pattern that the project needs to be added to ProjectInfo before it can be added via event